.github/workflows/ci.yml

```diff
@@ -98,6 +98,9 @@ jobs:
       - name: Install npm packages
         run: yarn install
 
+      - name: Install testrepo deps
+        run: cd rewatch/testrepo && yarn install
```

Comment:
Ideally `rewatch/testrepo` should be added to the yarn workspaces, then we wouldn't need this. We can look into that in a separate PR though.